src/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0 auto;
+  padding: 0 auto;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  position: relative;
```

Comment:
You can get rid of all `position: relative;` in this file. We don't want anything positioned this way here